src/ldp/auxiliary/CompositeAuxiliaryStrategy.ts

```diff
@@ -0,0 +1,58 @@
+import type { Representation } from '../representation/Representation';
+import type { RepresentationMetadata } from '../representation/RepresentationMetadata';
+import type { ResourceIdentifier } from '../representation/ResourceIdentifier';
+import type { AuxiliaryIdentifierStrategy } from './AuxiliaryIdentifierStrategy';
+import type { AuxiliaryStrategy } from './AuxiliaryStrategy';
+import type { MetadataGenerator } from './MetadataGenerator';
+import type { Validator } from './Validator';
+
+/**
+ * An {@link AuxiliaryStrategy} that provides its functionality through the combination of
+ * an {@link AuxiliaryIdentifierStrategy}, {@link MetadataGenerator} and {@link Validator}.
+ */
+export class CompositeAuxiliaryStrategy implements AuxiliaryStrategy {
```

Comment:
I just wonder if this is the right name, given that

> The composite pattern describes a group of objects that are treated the same way as a single instance of the same type of object.

Maybe

```suggestion
export class ComposedAuxiliaryStrategy implements AuxiliaryStrategy {
```

or so